samples/snippets/visualbasic/programming-guide/language-features/data-types/tuple-returns.vb

```diff
@@ -2,9 +2,9 @@
 Imports System.Globalization
 
 Public Module NumericLibrary
-    Public Function ParseInteger(value As String) As (Success As Boolean, Number As Int32)
+    Public Function ParseInteger(value As String) As (Success As Boolean, Number As Integer)
```

Comment:
The error abovefrom the snippets 5000 is from our new CI system. Historically, code was in a different system and was validated by an internal tool. When those samples were imported, the project files weren't added. Our CI system detects when code was changed and there wasn't a project to builld.  Adding a new .vbproj in this folder should fix the problem.